### PR TITLE
Add a link to the RHEL 7 to RHEL 8 post upgrade steps section

### PR DESCRIPTION
**Pull request by AkshayGadhaveRH**
The link to the post upgrade steps is missing in the upgrading RHEL 7 to RHEL 8 using LEAPP section. This results in some issues post upgrade (e.g. leapp setting the release version to 8.6). https://bugzilla.redhat.com/show_bug.cgi?id=2164027

* [X] I am familiar with the [contributing](https://github.com/theforeman/foreman-documentation/blob/master/CONTRIBUTING.md) guidelines.

Please cherry-pick my commits into:

* [ ] Foreman 3.5/Katello 4.7 (planned Satellite 6.13)
* [ ] Foreman 3.4/Katello 4.6 (EL8 only)
* [ ] Foreman 3.3/Katello 4.5 on EL7 & EL8 (Satellite 6.12 on EL8 only)
* [ ] Foreman 3.2/Katello 4.4 on EL7 & EL8
* [X] Foreman 3.1/Katello 4.3 on EL7 & EL8 (Satellite 6.11 EL7/8, orcharhino 6.1 on EL7, orcharhino 6.2 on EL7/8)
* For Foreman 3.0 or older, please create a separate PR.
* We do not accept PRs for Foreman 2.3 or older.

### CODE REVIEW

**Comment by AkshayGadhaveRH** [comment]
@evgeni added the link to the post-upgrade section for LEAPP.

File: guides/common/modules/proc_upgrading-project-or-proxy-in-place-using-leapp.adoc [hunk 1]
```diff
@@ -309,7 +309,9 @@ ifdef::katello[]
 ----
 endif::[]
 ifdef::satellite[]
-. Complete the post-upgrade steps described in https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/upgrading_from_rhel_7_to_rhel_8/verifying-the-post-upgrade-state-of-the-rhel-8-system_upgrading-from-rhel-7-to-rhel-8[Verifying the post-upgrade state of the RHEL 8 system] in the _Upgrading from RHEL 7 to RHEL 8_ guide.
+. Complete the post-upgrade steps described in https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/upgrading_from_rhel_7_to_rhel_8/performing-post-upgrade-tasks-rhel-7-to-rhel-8_upgrading-from-rhel-7-to-rhel-8[Performing post-upgrade tasks] in the _Upgrading from RHEL 7 to RHEL 8_ guide.
+
+. Verify the post-upgrade state of the system as described in https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/upgrading_from_rhel_7_to_rhel_8/verifying-the-post-upgrade-state-of-the-rhel-8-system_upgrading-from-rhel-7-to-rhel-8[Verifying the post-upgrade state of the RHEL 8 system] in the _Upgrading from RHEL 7 to RHEL 8_ guide.
```
Comment (evgeni) [inline guides/common/modules/proc_upgrading-project-or-proxy-in-place-using-leapp.adoc:314]: verify is chapter 6, perform is chapter 7, so I find it odd to link them the other way round.

So maybe have one step that goes "Verify the state as described in … and complete any outstanding post-upgrade steps as described in …"?

Comment (AkshayGadhaveRH) [reply guides/common/modules/proc_upgrading-project-or-proxy-in-place-using-leapp.adoc:318]: @evgeni my bad, I muddled up the order of the chapters. Fixed nbw.

File: guides/common/modules/proc_upgrading-project-or-proxy-in-place-using-leapp.adoc [hunk 2]
```diff
@@ -313,7 +313,7 @@ ifdef::katello[]
 ----
 endif::[]
 ifdef::satellite[]
-. Complete the post-upgrade steps described in https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/upgrading_from_rhel_7_to_rhel_8/verifying-the-post-upgrade-state-of-the-rhel-8-system_upgrading-from-rhel-7-to-rhel-8[Verifying the post-upgrade state of the RHEL 8 system] in the _Upgrading from RHEL 7 to RHEL 8_ guide.
+. Verify the post-upgrade state of the system as described in https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/upgrading_from_rhel_7_to_rhel_8/verifying-the-post-upgrade-state-of-the-rhel-8-system_upgrading-from-rhel-7-to-rhel-8[Verifying the post-upgrade state of the RHEL 8 system] in the _Upgrading from RHEL 7 to RHEL 8_ guide and complete any remaining post-upgrade steps described in https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/upgrading_from_rhel_7_to_rhel_8/performing-post-upgrade-tasks-rhel-7-to-rhel-8_upgrading-from-rhel-7-to-rhel-8[Performing post-upgrade tasks] in the _Upgrading from RHEL 7 to RHEL 8_ guide.
```
Comment (evgeni) [inline guides/common/modules/proc_upgrading-project-or-proxy-in-place-using-leapp.adoc:316]: Both chapters are in the same guide, and adding "in the _Upgrading from RHEL 7 to RHEL 8_ guide" twice kinda makes this sentence longer and bulkier, without adding any real value. Can we somehow shorten this to one mention of the guide name?

**Review by evgeni — APPROVED** [review]
Happy with the content, but would be even happier if we could shorten the sentence a bit ;)

**Comment by evgeni** [comment]
`ovirt.org` is down, so the link checker fails.

**Review by evgeni — APPROVED** [review]
perfect! (will need to be squashed when merging)

**Comment by adamlazik1** [comment]
Merged to 3.1.
